CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+## Your new git-extra command should support...
+
+* Should support Mac and Linux(You may need to browse their man page)
+* Should support Bash 3 or newer(If you are not sure, [Bash versions](http://tldp.org/LDP/abs/html/bash2.html))
```

Comment:
I don't actually think we will have much difficulties supporting Bash 3.0 (at least that's the impression I got from scanning over the changelogs), but it may be difficult for people to test it. I tried building Bash 3.0 on OSX, and I was unable to, even with all the patches.

Now, the default shell on my OSX El Capitan is 3.2 so that one shouldn't be hard to get hold of.

Should we say `Bash >= 3.2`?